addons/l10n_cl/views/l10n_latam_document_type_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="l10n_cl_latam_document_type_view" model="ir.ui.view">
+        <field name="name">l10n.cl.latam.document.type.view</field>
+        <field name="model">l10n_latam.document.type</field>
+        <field name="inherit_id" ref="l10n_latam_invoice_document.view_document_type_form"/>
+        <field name="arch" type="xml">
+            <field name="sequence" position="attributes">
+                    <attribute name="widget">handle</attribute>
```

Comment:
Why?  Better to change in the l10n_latam_invoice_document then.

Reply:
Because with this widget the user could change the order of the document types are showing in the invoice. The current code doesn't allow it.

Reply:
I can remove it if you want but I think it's a good idea to improve the usability of the users.

Reply:
I don't think it belongs here, but it does not bother me that much.  Could be good to check in the future with @jjscarafia if we should not put it in l10n_latam_invoice_document already.  There is also the risk that the user changes the order by accident and when you upgrade, the changes could be lost.